MekHQ/src/mekhq/campaign/market/ShoppingList.java

```diff
@@ -37,58 +36,66 @@
 import mekhq.campaign.unit.UnitOrder;
 import mekhq.campaign.work.IAcquisitionWork;
 
 /**
- *  A list of IAcquisitionWork
+ * A list of IAcquisitionWork
  *
- *  When a new acquisition is requested (via the parts store or the acquisition tab), we
- *  iterate through this list and look for the MissingPart.getNewPart that matches
- *  the desired part. Here are the possible outcomes:
+ * When a new acquisition is requested (via the parts store or the acquisition tab), we
+ * iterate through this list and look for the MissingPart.getNewPart that matches
+ * the desired part. Here are the possible outcomes:
  *
- *  - We find it, but we cannot check today, so we add the quantity requested
- *  - We don't find it, we immediately check and add to the list if we fail
+ * - We find it, but we cannot check today, so we add the quantity requested
+ * - We don't find it, we immediately check and add to the list if we fail
  *
- *  On Campaign.newDay, we also cycle through the list and check any items that have no
- *  more days to wait for the next check.
+ * On Campaign.newDay, we also cycle through the list and check any items that have no
+ * more days to wait for the next check.
  *
- *  Checking procedure
- *  Using a while loop, we keep checking using an acquisition roll until we fail or we hit
- *  zero quantity. If we hit zero quantity, then we can remove the item. If we fail, then
- *  we reset the dayCounter to the max.
+ * Checking procedure
+ * Using a while loop, we keep checking using an acquisition roll until we fail or we hit
+ * zero quantity. If we hit zero quantity, then we can remove the item. If we fail, then
+ * we reset the dayCounter to the max.
  *
- *  We also now only use one person to make all checks. We allow the user to set the skill
- *  and other options for who makes the check in the campaign options.,
+ * We also now only use one person to make all checks. We allow the user to set the skill
+ * and other options for who makes the check in the campaign options.,
  *
- *   Do we use a separate shopping list for new units?
+ * Do we use a separate shopping list for new units?
  */
 public class ShoppingList implements MekHqXmlSerializable {
-
+    //region Variable Declarations
     private List<IAcquisitionWork> shoppingList;
+    //endregion Variable Declarations
 
+    //region Constructors
     public ShoppingList() {
-        shoppingList = new ArrayList<IAcquisitionWork>();
+        setShoppingList(new ArrayList<>());
     }
-    
-    public ShoppingList(List<IAcquisitionWork> items) {
-        shoppingList = items;
+
+    public ShoppingList(List<IAcquisitionWork> shoppingList) {
+        setShoppingList(shoppingList);
     }
-    
+
     public ShoppingList(IAcquisitionWork item) {
-        shoppingList = new ArrayList<IAcquisitionWork>();   
-        shoppingList.add(item);
+        setShoppingList(new ArrayList<>());
+        getShoppingList().add(item);
```

Comment:
Do we want to have a mutable list returned? We've been paring those down in other places.

Reply:
Missed my last commit cleaning up the old code seen here. There were no actual uses of this, so I've removed the code in question.